src/axom/sidre/core/Group.cpp

```diff
@@ -2032,20 +2149,22 @@ void Group::save(const hid_t& h5_id,
     exportTo(n["sidre"], attr);
     createExternalLayout(n["sidre/external"], attr);
     n["sidre_group_name"] = m_name;
-    conduit::relay::io::hdf5_write(n, h5_id);
+    checkConduitCall([&] { conduit::relay::io::hdf5_write(n, h5_id); });
   }
   else if(protocol == "conduit_hdf5")
   {
     Node n;
     createNativeLayout(n, attr);
     n["sidre_group_name"] = m_name;
-    conduit::relay::io::hdf5_write(n, h5_id);
+    checkConduitCall([&] { conduit::relay::io::hdf5_write(n, h5_id); });
   }
   else
   {
     SLIC_ERROR(SIDRE_GROUP_LOG_PREPEND << "Invalid protocol '" << protocol
                                        << "' for save with hdf5 handle.");
   }
```

Comment:
As above, the code should return false if an invalid protocol was passed.
Please also check the other similar cases.
(Note `SLIC_ERROR` can be configured to not call `exit`).